aemsites/ingredion
# feat: Back To Top Button

This PR introduces a "Back to Top" button across all pages. The button appears after scrolling down a specified threshold and includes smooth scrolling functionality. It is hidden on mobile devices and short pages

Fix #51 

Testing Instructions
Scroll down a page to ensure the button appears after crossing the scroll threshold.
Click the button and verify it smoothly scrolls to the top.
Test the button's responsiveness (does not appear on mobile screens).
Check that it doesn't appear on short pages with insufficient scroll height.
